server/services/offender/index.js

```diff
@@ -270,5 +270,5 @@ const createOffenderService = (
 };
 
 module.exports = {
-  createPrisonApiOffenderService: createOffenderService,
+  createPrisonAndIncentivesApiOffenderService: createOffenderService,
```

Comment:
Slightly clunky name so open to suggestions 😄

Reply:
`createOffenderService`